anime-list-master.xml

```diff
@@ -37555,7 +37567,7 @@
   <anime anidbid="13691" tvdbid="102261" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Zoku Owarimonogatari</name>
     <mapping-list>
-      <mapping anidbseason="1" tvdbseason="0">;1-33;2-33;3-34;4-35;5-36;6-37;7-38;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;1-0;2-33;3-34;4-35;5-36;6-37;7-38;</mapping>
```

Comment:
Worked for me
![image](https://i.imgur.com/lPg0ANl.png)

Reply:
Will you try the following tests?
- delete episode 1, then refresh metadata
- make similar tests (with and without episode 1) but as [anidb-13691]

Reply:
Episode 1 disappeared as expected. The title and thumbnail are TVDB's probably because that's my primary provider for most fields on HAMA settings
![image](https://user-images.githubusercontent.com/7104931/194728345-e8c81f44-94ff-4517-95fa-50bede00388f.png)